pdm/cli/commands/run.py

```diff
@@ -84,7 +86,9 @@ def _run_command(
         else:
             os.execv(expanded_command, expanded_args)
 
-    def _normalize_script(self, script):
+    def _normalize_script(
+        self, script: Any
+    ) -> Tuple[str, Union[Array, String], Union[Dict[str, String], Dict[str, Table]]]:
```

Comment:
```suggestion
    ) -> Tuple[str, Union[List[str], str], Dict[str, Union[str, dict]]:
```
Avoid using internal types from `tomlkit`

Reply:
Fixed here. I also removed all internal types from tomlkit from codes.

[`cec10b8` (#354)](https://github.com/pdm-project/pdm/pull/354/commits/cec10b86b17254eebcd050e47ae8bda258b65a27)